inductiva/resources/machines_base.py

```diff
@@ -132,20 +133,72 @@ def from_api_response(cls, resp: dict):
         machine_group = cls(
             machine_type=resp["machine_type"],
             data_disk_gb=resp["disk_size_gb"],
-            provider=resp["provider_id"],
             register=False,
         )
         machine_group._id = resp["id"]
+        machine_group.provider = resp["provider_id"]
         machine_group._name = resp["name"]
         machine_group.create_time = resp["creation_timestamp"]
         machine_group._started = True
 
         return machine_group
 
-    def start(self, **kwargs):
+    def update_termination_timers(self,
```

Comment:
this is a helper method and can be hidden.

```suggestion
    def _update_termination_timers(self,
```